experiment/test_config.yaml

```diff
@@ -690,7 +690,7 @@ nodeCommon:
     envs:
     - CLEANUP=true
     - GCE_ZONE=us-central1-f
-    - KUBELET_ARGS=--cgroups-per-qos=true --cgroup-root=/
+    - KUBELET_ARGS="--cgroups-per-qos=true --cgroup-root=/"
```

Comment:
`kubernetes_kubelet.py` scenario does not support passing `--test_args` and its usage of the envs requires quotes. I planed to have the test running first and then migrate those envs to flags, as what we've been doing for e2e scenario. That's why I temporarily make an exception for node-e2e tests in `bootstrap_test.py` in this PR. But I think I can also do this now.